tools/generate_tx_models.py

```diff
@@ -0,0 +1,189 @@
+"""Script to generate transaction models from rippled source code."""
```

Comment:
@mvadari Can you write a test for this that includes a sample file that is parsed and sample output?

Reply:
I thought about that but it's pretty difficult to do, since you're editing source code.

Reply:
@mvadari I don't follow. I think you could write it using this: https://docs.python.org/3/library/ast.html

Reply:
The way this script works is that it basically edits the xrpl-py source code to inject the transaction models and details. Writing a test with AST would be more difficult than it's worth IMO.